Создал список user stories для приложения "lazy planing"

Heriuple opened the pull request:
Создал список пользовательских историй, с директориями

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-1/READMY.md to docs/Technical/UserStories/Story-01/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-2/READMY.md to docs/Technical/UserStories/Story-02/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-3/READMY.md to docs/Technical/UserStories/Story-03/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-4/READMY.md to docs/Technical/UserStories/Story-04/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-5/READMY.md to docs/Technical/UserStories/Story-05/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-6/READMY.md to docs/Technical/UserStories/Story-06/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-7/READMY.md to docs/Technical/UserStories/Story-07/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-8/READMY.md to docs/Technical/UserStories/Story-08/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-9/READMY.md to docs/Technical/UserStories/Story-09/READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple commented:
Прошу прощения за обилие комитов, работал через веб версию и требовалось привести документы к общему значению.

YuriSilenok reviewed (approved):
У нас будут стори по запуску таймера на задачу?

Heriuple commented:
Да, я внесу её сейчас

Heriuple committed:
Create READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-11/READMY.md to Docs/Technical/UserStories/Story-11/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-10/READMY.md to Docs/Technical/UserStories/Story-10/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-01/READMY.md to Docs/Technical/UserStories/Story-01/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-02/READMY.md to Docs/Technical/UserStories/Story-02/READMY.md

Heriuple committed:
Update and rename docs/Technical/UserStories/Story-03/READMY.md to Docs/Technical/UserStories/Story-03/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-04/READMY.md to Docs/TechnicalUserStories/Story-04/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-05/READMY.md to Docs/Technical/UserStories/Story-05/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-06/READMY.md to Docs/Technical/UserStories/Story-06/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-07/READMY.md to Docs/Technical/UserStories/Story-07/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-08/READMY.md to Docs/Technical/UserStories/Story-08/READMY.md

Heriuple committed:
Rename docs/Technical/UserStories/Story-09/READMY.md to Docs/Technical/UserStories/Story-09/READMY.md

Heriuple committed:
Rename Docs/TechnicalUserStories/Story-04/READMY.md to Docs/Technical/UserStories/Story-04/READMY.md

YuriSilenok commented on Docs/Technical/UserStories/Story-11/READMY.md:
```diff
@@ -0,0 +1 @@
+[Story-11] Пользователь может запустить таймер по отсчёту времени затраченной на задачу.
```
Решетки не хватает в заголовке

YuriSilenok commented on Docs/Technical/UserStories/Story-09/READMY.md:
```diff
@@ -0,0 +1 @@
+# [Story-09] Пользователь получает доступ к списку запланированных задач на сегодня, с прописанным кол-вом времени на них
```
Не понял в чем отличие от `# [Story-07] Пользователь получает равномерно распределенную нагрузку задач` ?

Heriuple replied:
ВЫ говорили, что в базовой версии требуется прописать, текущей день с задачами и завтрашний. А 7 стори, отвечает именно за ту фишку, о которой мы говорим, равномерном распределении.

YuriSilenok replied:
У нас проблема понимания.
Когда мы смотрим на список задач на сегодня, разве мы не видим  их сразу в распределенном виде?
И что-то я не помню относительно стори для отображения списка задач на завтра. Давай ее не будем делать, она не входит в минимальный функционал.

Heriuple replied:
Хорошо, я отмечу стори про завтра как расширяющую функционал. Сейчас постараюсь написать, что я вкладываю в каждую из этих стори. 9 стори отвечает только за отображение задач в списке сделать на сегодня, и без 7 стори, равномерного распределения  не будет, будет так-же отображаться как в том приложении, которое вы мне показали в качестве примера. Т. е. стори-7 отвечает за алгоритм распределения, а 9 за список уже распределенных задач.

YuriSilenok reviewed (commented):
Думаю у нас не хватает 
- настроек интервалов времени
- настройка о том, нужно ли останавливать счетчик времени, когда кончается выделенное время на задачу
- звуковые сигналы по истечению интервала времени

Heriuple commented:
принято, добавлю

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Create READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Rename Docs/Technical/UserStories/Story-10/READMY.md to Docs/Technical/UserStories/Story-10(не в ходит в базовый функционал)/READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Update READMY.md

Heriuple committed:
Разработал UserStory [Story-05]

Разработал UserStory [Story-05] Пользователь удаляет созданный ранее проект

Heriuple committed:
Update READMY.md

Heriuple committed:
Разработал UserStory [Story-08]

Разработал UserStory [Story-07] В которой прописано что пользователь может прописать кол-во томатов для своей задачи

Heriuple committed:
Разработал UserStory [Story-09]

Разработал UserStory [Story-06] В которой указанно, что пользователь имеет отдельное окно задач на сегодня.

Heriuple committed:
Разработал UserStory [Story-11]

Разработал UserStory [Story-11]  В которой указано что пользователь может запустить таймер по отсчёту времени томата.

Heriuple committed:
Разработал UserStory [Story-12]

Разработал UserStory [Story-12] Пользователь может изменять длину томатов, длину отдыха и кол-во томатов между большим промежутком.

Heriuple committed:
Разработал UserStory [Story-13]

Разработал UserStory [Story-13] В котором описано получение звукового сигнала пользователем по завершению томата и отдыха

Heriuple committed:
Разработал UserStory [Story-14]

Разработал UserStory [Story-14] В котором указана возможность пользователя решить может ли приложение останавливать таймер по завершению всех томатов по задаче на сегодня или нет.

YuriSilenok committed:
Update READMY.md

Heriuple committed:
Merge pull request #6 from Students-of-the-city-of-Kostroma/task-3

Разработал UserStory [Story-01]

Heriuple committed:
Merge pull request #20 from Students-of-the-city-of-Kostroma/task-5

Разработал UserStory [Story-03]

Heriuple committed:
Merge pull request #21 from Students-of-the-city-of-Kostroma/task-6

Разработал UserStory [Story-04]

Heriuple committed:
Merge pull request #22 from Students-of-the-city-of-Kostroma/task-7

Разработал UserStory [Story-05]

Heriuple committed:
Merge pull request #23 from Students-of-the-city-of-Kostroma/task-8

Разработал UserStory [Story-06]

Heriuple committed:
Merge pull request #8 from Students-of-the-city-of-Kostroma/task-4

Разработал UserStory [Story-02]

Heriuple committed:
Merge pull request #25 from Students-of-the-city-of-Kostroma/task-10

Разработал UserStory [Story-08]

Heriuple committed:
Merge pull request #26 from Students-of-the-city-of-Kostroma/task-11

Разработал UserStory [Story-09]

Heriuple committed:
Merge pull request #27 from Students-of-the-city-of-Kostroma/task-12

Разработал UserStory [Story-11]

Heriuple committed:
Merge pull request #28 from Students-of-the-city-of-Kostroma/task-13

Разработал UserStory [Story-12]

Heriuple committed:
Merge pull request #29 from Students-of-the-city-of-Kostroma/task-14

Разработал UserStory [Story-13]

Heriuple committed:
Merge pull request #30 from Students-of-the-city-of-Kostroma/task-15

Разработал UserStory [Story-14]
